requirements-test.txt

```diff
@@ -0,0 +1,7 @@
+coveralls>=1.8.2,<2.0.0
+pytest<6.0.0
+pytest-cov<3.0.0
+pytest-mock<4.0.0
+pytest-pycodestyle<3.0.0
+pytest-asyncio>=0.15.1
```

Comment:
spoiler alert, get `pytest-asyncio` outa here (it's evil and will crash pytest on travis)